### ISSUE TITLE
Create relationship items inline in Keystone 4.0 beta 4 fails

### ISSUE THREAD

**Issue by VadimShvetsov** [keystonejs/keystone#3786]
I need to add relationship item within other model. I was found this PR [#2093](https://github.com/keystonejs/keystone/pull/2093#issuecomment-171737666) and was tried to use this.

### Steps to reproduce the behavior

```
Post.add({
	name: { type: String, required: true, index: true },
	state: { type: Types.Select, options: 'draft, published, archived', default: 'draft', index: true },
	author: { type: Types.Relationship, ref: 'User', index: true },
	publishedDate: { type: Types.Date, index: true },
	image: { type: Types.File, storage: localStorage },
	content: {
		brief: { type: Types.Html, wysiwyg: true, height: 150 },
		extended: { type: Types.Html, wysiwyg: true, height: 400 },
	},
	categories: { type: Types.Relationship, ref: 'PostCategory', createInline: true, many: true },
	tags: { type: Types.Relationship, ref: 'Tag', createInline: true, many: true },
});
```

### Expected behavior
When I press "+" at Admin UI on the right of field I must get a modal for add new item.

### Actual behavior
When I press "+" at Admin UI I can see modal is about 100ms then page get reloaded.

And the UI displays too short string, is it expected behavior?
<img width="291" alt="2016-12-12 17 09 19" src="https://cloud.githubusercontent.com/assets/16336572/21103014/b23e5728-c091-11e6-84a6-ae01d3a32481.png">

**Comment by VadimShvetsov** [comment]
@pavethiran Thanks for so fast fixing. But anyway UI stands wrong but modal works properly.

<img width="385" alt="2016-12-15 13 04 47" src="https://cloud.githubusercontent.com/assets/16336572/21219813/5e8cd694-c2c7-11e6-87f7-2be6f12d29c1.png">

**Comment by jstockwin** [comment]
@VadimShvetsov Can this be closed? I think the display issue you mention was fixed in the above #3846?

**Comment by VadimShvetsov** [comment]
@jstockwin Thanks, for sure!
